eosdatadistribution/producers/soma_subscriptions.py

```diff
@@ -25,10 +25,15 @@
 
 from gi.repository import GLib
 
-from eos_data_distribution.soma_subscription_fetcher import Fetcher
+from eosdatadistribution.soma_subscription_fetcher import Fetcher
```

Comment:
so our naming scheme is a bit of the oposite, i.e. $what.py -> {producer,consumer} it usually makes more sense.